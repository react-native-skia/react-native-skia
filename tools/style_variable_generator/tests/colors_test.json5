{
  options: {
    CSS: {
      prefix: 'cros'
    },
    proto: {
      field_name: 'test_colors',
      field_id: 2,
    },
  },
  colors: {
    text_color_primary: {
      light: "$google_grey_900",
      dark: "$white",
      debug: "$toggle_color",
    },
    toggle_color: {
      light: "rgba($text_color_primary.rgb, 0.1)",
      dark: "rgba($text_color_primary.rgb, $disabled_opacity)",
      debug: "$white",
    },
    bg_color_elevation_1: {
      light: "$white",
      dark: "blend(rgba($white.rgb, 0.04), $google_grey_900)",
    }
  },
  opacities: {
    disabled_opacity: 0.38,
    reference_opacity: {
        light: '$disabled_opacity',
        dark: 1,
    },
  },
}
