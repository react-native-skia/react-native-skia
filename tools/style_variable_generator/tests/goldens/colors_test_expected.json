{
    "ref_tokens": [
        {
            "token_name": "google_grey_900",
            "css_variable": "--google-grey-900"
        },
        {
            "token_name": "text_color_primary",
            "css_variable": "--cros-text-color-primary"
        },
        {
            "token_name": "toggle_color",
            "css_variable": "--cros-toggle-color"
        },
        {
            "token_name": "bg_color_elevation_1",
            "css_variable": "--cros-bg-color-elevation-1"
        }
    ],
    "sys_tokens": []
}
